sharding/db.go

```diff
@@ -0,0 +1,46 @@
+package sharding
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type shardKV struct {
+	kv map[common.Hash][]byte
+}
+
+func makeShardKV() *shardKV {
+	return &shardKV{kv: make(map[common.Hash][]byte)}
+}
+
+func (sb *shardKV) Get(k common.Hash) ([]byte, error) {
+	v, ok := sb.kv[k]
+	fmt.Printf("Map: %v\n", sb.kv)
+	fmt.Printf("Key: %v\n", k)
+	fmt.Printf("Val: %v\n", sb.kv[k])
+	fmt.Printf("Ok: %v\n", ok)
```

Comment:
Remove all of this printing. If you feel that you really need it, use ethereum log with the appropriate level (debug).